doc/source/rllib/rllib-learner.rst

```diff
@@ -0,0 +1,438 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. |tensorflow| image:: images/tensorflow.png
+    :class: inline-figure
+    :width: 16
+
+.. |pytorch| image:: images/pytorch.png
+    :class: inline-figure
+    :width: 16
+
+
+Learner (Alpha)
+==================
+
+:py:class:`~ray.rllib.core.learner.learner.Learner` allows you to abstract the training 
```

Comment:
Can we add the API docs? `Learner` API references, right now are not recognized.

Reply:
done